### PR TITLE
ZEPPELIN-3099 Livy Interpreter doesn't support German Special Character - Encoding Issue

### PR DESCRIPTION
**Pull request by Akhilsnaik**
### What is this PR for?
ZEPPELIN-3099 Livy Interpreter doesn't support German Special Character - Encoding Issue

### What type of PR is it?
BUG FIX for : ZEPPELIN-3099

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-3099

### How should this be tested?
test with German Special Characters like ß, ü, Ü, ä, Ä, ö, Ö .

### CODE REVIEW

**Comment by Akhilsnaik** [comment]
@zjffdu  @prabhjyotsingh  

the PR : 2850 got messed up due to GIt remote merge. 
I have implemented the review comments mentioned in PR 2850 
I am raising a new PR for the issue ZEPPELIN-3099
Please help me review this.

File: livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java [hunk 1]
```diff
@@ -379,6 +379,15 @@ public void testPySparkInterpreter() throws InterpreterException {
     } catch (Exception e) {
       e.printStackTrace();
     }
+        //test special characters
+    try {
```
Comment (zjffdu) [inline livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java]: Please remove the try catch as the exception here is ignored. Could you also help to remove the try catch in the above unicode test ?

Comment (Akhilsnaik) [reply livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java]: Hi @zjffdu  ,

I have implemented the review comment , and commited along with this PR, Please feel free to review.

**Comment by zjffdu** [comment]
LGTM, will merge after CI pass

**Pull request closed without merging**